src/layer/GeoJSON.js

```diff
@@ -240,24 +240,25 @@ export function coordsToLatLngs(coords, levelsDeep, _coordsToLatLng) {
 	return latlngs;
 }
 
-// @function latLngToCoords(latlng: LatLng): Array
+// @function latLngToCoords(latlng: LatLng, precision?: Number): Array
 // Reverse of [`coordsToLatLng`](#geojson-coordstolatlng)
-export function latLngToCoords(latlng) {
+export function latLngToCoords(latlng, precision) {
+	precision = precision || 6;

```

Comment:
this way you cannot round to 0 decimals. When (if) my pr #5492 lands, you can skip this line of code.